### PR TITLE
Add deprecation notice

### PR DESCRIPTION
**Pull request by Jako**

### CODE REVIEW

File: en/extras/addheaderfiles/index.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ _old_id: "1691"
 _old_uri: "revo/addheaderfiles"
 ---
 
+!!! warning "Deprecated"
```
Comment (Mark-H) [inline en/extras/addheaderfiles/index.md]: Do we have this syntax? I've never seen it before.

We do support `note` in the metadata to put an obvious alert at the top.

Comment (Jako) [reply en/extras/addheaderfiles/index.md:7]: It depends on the Daux version: https://daux.io/Features/Admonition.html

Comment (Jako) [reply en/extras/addheaderfiles/index.md]: Sorry, it seems that the DocsApp does not use modern markdown code as [Daux.io](https://daux.io/Features/Admonition.html) and [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/reference/)

Comment (Mark-H) [reply en/extras/addheaderfiles/index.md]: Interesting, first I've heard of that. If you want to [open an issue here](https://github.com/modxorg/DocsApp/issues) maybe we can add it.

**Pull request merged**